app/bundles/CoreBundle/Tests/Functional/Doctrine/Paginator/SimplePaginatorTest.php

```diff
@@ -18,18 +16,10 @@ class SimplePaginatorTest extends MauticMysqlTestCase
      * @var array<string,mixed>
      */
     protected array $clientOptions = ['debug' => true];
-    private DebugStack $logger;
 
     protected function setUp(): void
     {
         parent::setUp();
-
-        $logger = self::$container->get('doctrine.dbal.logger.profiling.default');
-        assert($logger instanceof DebugStack);
-        $logger->queries      = [];
-        $logger->currentQuery = 0;
-
-        $this->logger = $logger;
```

Comment:
@fedys could you please see if there is a way to do this in the new Doctrine version? Me and @mabumusa1 failed to  find a way.

Reply:
I discussed this with Fedy. Asserting the queries are not that important in this test. The output is being asserted and that matters. We can get back to solving the query logging later but it should not block this PR and M5 release.

Reply:
Resolved in https://github.com/mautic/mautic/commit/f7debd8cb29156ff74e14df0be29a98ad67cd676